src/SprykerEco/Zed/Braintree/Business/Payment/Transaction/RefundItemsTransaction.php

```diff
@@ -122,7 +122,7 @@ protected function refund()
      */
     protected function getAmount(): float
     {
-        return $this->moneyFacade->convertIntegerToDecimal($this->transactionMetaTransfer->getRefundAmount());
+        return $this->moneyFacade->convertIntegerToDecimal((int)$this->transactionMetaTransfer->getRefundAmount());
```

Comment:
`getRefundAmount` is already float according to transfers definition

Reply:
While transfer definition implies that the refund amount is float, the actual type of the data it contains actually seems to be int. This can be tracked in code where the amount property for this transfer is set. Also `convertIntegerToDecimal` would fail if called with a non-int argument, so the amount type is definitely int. 

Since changing a transfer property type is a BC break, it seems to be reasonable to leave this as is for now. (The transfer property was not introduced in this PR)